src/app/components/work-item-detail/work-item-detail.component.html

```diff
@@ -1,19 +1,28 @@
-<div [@slideInOut]="panelState" class="detail-panel">
-  <div class="detail-container" *ngIf="workItem" id="workItemDetail_Wrapper">
+<div [@slideInOut]="panelState" class="f8-quick-preview">
+  <div class="width-100" *ngIf="workItem" id="workItemDetail_Wrapper">
     <form *ngIf="active"
           #almWorkItemDetailForm="ngForm"
           class="form-horizontal"
           id="wi-detail-form">
       <fieldset>
-        <div class="detail-panel-header">
-          <div class="form-group col-xs-12 col-sm-12 col-md-12 mobile-bar">
-            <div class="dropdown-kebab-pf detail-type-dropdown col-sm-8">
+        <div class="f8-quick-preview__header">
```

Comment:
for Element we are using two underscores. As one underscore we are using for id naming converntion